### ISSUE TITLE
Investigate broken prefit errors in YieldsTable

### ISSUE THREAD

**Issue by longjon929** [histfitter/histfitter#160]
Someone report that with v1.2.0 and 1.2.2 that the prefit errors in yields tables are wonky.  It could be related to the v6.26 ROOT changes.

**Comment by longjon929** [comment]
It's on the workspace creation side between 1.1 and master.  A workspace made with 1.1 works when read in in either tag.

**Comment by longjon929** [comment]
I'm not sure, but given the post fit errors are ok I would think it's not a pdf name issue?

**Comment by jeanyvesb9** [comment]
Ok. Did you see the problem with any other workspace? Because I can't access Simran's

**Comment by longjon929** [comment]
Yes, I can reproduce it with the analysis/tutorial/MyUserAnalysis.py config
and running `YieldsTable.py -c UserRegion -w
results/MyUserAnalysis/SPlusB_combined_NormalMeasurement_model_afterFit.root
-s Sig,Bkg -o yields.tex -b`

**Comment by jeanyvesb9** [comment]
The error seems to be related to the luminosity uncertainty. On the broken workspaces, it's set to 100%

**Comment by jeanyvesb9** [comment]
Yes, it is definitely related to the luminosity.

In Utils::resetErrors, a constant value of 1.0 is returned for the "lumiConstraint" sigma (https://github.com/histfitter/histfitter/blob/master/src/Utils.cxx#LL2618C67-L2618C67). I'm investigating whether this is a bug in ROOT itself, or whether we need to update how we're reading it.

**Comment by jeanyvesb9** [comment]
When HistFactory runs, it defines three different objects in the workspace to take into account the Luminosity (https://github.com/root-project/root/blob/2e2309327ec21930378bf859bc854280b4e36f8e/roofit/histfactory/src/HistoToWorkspaceFactoryFast.cxx#L795-L807):

- RooRealVar "nominalLumi", with a minimum value of 0, and a maximum value of nominal + 10 * sigma,
- RooRealVar "Lumi", with a minimum value of 0 and a maximum value of 10 * nominal,
- RooGausian "lumiConstraint", with its x variable set to "Lumi", its mean set to "nominalLumi", and a fixed std deviation set to sigma.

The way we obtained the luminosity uncertainty in HistFitter was very convoluted: we explicitly selected the first server of "lumiConstraint", which on ROOT <= v6.26 was sigma, and then we got its value. On ROOT v6.28, the order of the servers was changed, and now the RooGausian's sigma is last. I didn't find exactly which ROOT commit is responsible for this change, but nonetheless, there's a very simple fix:

`sigma = lumiConstr->getSigma().getVal()`

I'm looking into any other instances RooGaussian -> server -> value that could also be buggy now/in the near future, and could be replaced by this simplified approach. I'll have a PR ready soon.
